src/test/java/tests/FilterTest.java

```diff
@@ -0,0 +1,33 @@
+package tests;
+
+import entities.*;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.ApplePage;
+import pages.DominoPage;
+import pages.MainPage;
+import pages.SmartphonePage;
+import testData.Catalogue;
+
+public class FilterTest extends BaseTest {
+    @Test(description = "TC-5, Test to choose a manufacturer in the filter")
+    public void chooseManufacturerTest() {
+        ApplePage applePage = new MainPage()
+                .openPage()
+                .clickOnCatalogue()
+                .selectCatalogueDetail(SmartphonePage.class, Icons.ELECTRONICS, Electronics.MOBILE_PHONES, CatalogueOptions.SMARTPHONES)
```

Comment:
I think it would make sense to move class to the last argument because that's the thing we need for technical impl but the other 3 are the ones which show actual business logic

Reply:
understood